charts/trino/values.yaml

```diff
@@ -75,3 +75,12 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
```

Comment:
Should this default to `false` instead? It's not used in the chart by-default yet?